coreos-postinst

```diff
@@ -238,6 +242,10 @@ then
     cat > "${DOCKER_FLAG_PATH}" <<< yes
 fi
 
+tee_journal() {
+    cat - | tee >(cat - | systemd-cat -t coreos-postinst)
```

Comment:
@bgilbert that's my take at "We need to do a better job of surfacing update failures to the user". The user would consume this as `journalctl -t coreos-postinst`. Not sure if this is what you had in mind, and if it actually improves usability.

Reply:
It's better than nothing.  Ideally we'd put update failures e.g. in the MOTD and report them to an aggregation server somewhere.